stable/app/values.yaml

```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository:
+  repository: ~
```

Comment:
What is the reason for this change?

Reply:
just to explicitly show null value. And also if for some reason below lines are accidentally indented, `repository` won't be treated as an object.

Reply:
Got it. Thanks.